modules/instrument_builder/jsx/react.questions.js

```diff
@@ -0,0 +1,462 @@
+QuestionText = React.createClass({
```

Comment:
Now that I've went through the rest of the code and gotten here again, I'm still not certain what the distinction is between "questions" and "elements"